lib/modules/apostrophe-images/index.js

```diff
@@ -79,7 +79,7 @@ module.exports = {
   construct: function(self, options) {
     const titleField = _.find(options.addFields, {name: 'title'});
     if (titleField) {
-      titleField.help = 'This field used for the default image widget\'s "alternative text," read by assistive devices.';
+      titleField.help = 'This field is used for the default image widget\'s "alternative text," read by assistive devices.';
```

Comment:
Sorry to bother you again, but I am not sure if the `,` should be inside the quotes, or outside? For the feature/improvement in general, I have no opinion, I am just checking the text here.

Reply:
I personally default to AP style, which would put it inside. I'm not the most consistent, admittedly, since this is one of many places where the English grammar is regularly abused.